src/middlewared/middlewared/main.py

```diff
@@ -1122,6 +1124,13 @@ async def _run_in_conn_threadpool(self, method, *args, **kwargs):
         """
         return await self.run_in_executor(self.__ws_threadpool, method, *args, **kwargs)
 
+    def __terminate_orphaned_multiprocessing(self):
```

Comment:
Given that we are now killing pg, do we still need this?

I feel somewhat unsure about killing generic python processes like that, which we are not 100% being leftovers from middleware.

Reply:
These processes will be left alive if middleware was killed by OOM-killer on Linux (or other SIGKILL).

I think it's safe to kill them considering we only do so when process parent PID is 1 which won't be true if a multiprocessing python program that created them is still alive.

Reply:
Just dont think its our job to handle that if this is the case, sounds more like a python multiprocessing issue?